nvtabular/dask/io.py

```diff
@@ -370,3 +388,14 @@ def to_ddf(self, columns=None):
         return dask_cudf.read_csv(
             self.paths, names=self.names, chunksize=self.part_size, **self.csv_kwargs
         )[columns]
+
+    def to_iter(self, columns=None):
+        part_mem_fraction = self.part_size / cuda.current_context().get_memory_info()[1]
+        itr = GPUDatasetIterator(
+            self.paths,
+            engine="csv",
+            gpu_memory_frac=part_mem_fraction,
+            names=self.names,
```

Comment:
you don't need all the code to add `names` as a member variable.  This line could just be: `names=self.csv_kwargs.get("names", None)`